docs/guide/ecosystem/documentation.md

```diff
@@ -66,39 +66,53 @@ Below is a list of popular documentation generators, that are not connected to V
 
 ## Component workbenches
 
-A workbench is a development environment for UI components. It allows developers to setup interactive examples of how a UI component can be used. Simply put, you can setup a live demo of a Vue component by passing different props to it, creating a working sandbox.
+A workbench is a development environment for UI components. It allows developers to setup interactive examples of how a UI component can be  used. One can setup a live demo of a Vue component by passing different props to it. Perfect to show your designer or for you to test the result of your work
 
-This is very powerful when working in big teams with lots of custom UI components. A real rendered component with usage example can be easier to understand than just a text example.
+Workbenches are very powerful when working in big teams with lots of components. They allow other developers to discover how components work by tweaking them. Even better, these showcases can turn into static websites.
 
-### Storybook
+### Storybook 🌟
```

Comment:
I like this, but it generates a super weird anchor link. Maybe I should think of a better way. Lets remove it for now.

Reply:
removed